internal/cli/clients/client.go

```diff
@@ -1030,16 +1023,15 @@ func (c *EpinioClient) Target(org string) error {
 		WithStringValue("Name", org).
 		Msg("Targeting organization...")
 
-	// todo: fix, remove, move to server
-	details.Info("validate")
-	err := c.ensureGoodOrg(org, "Unable to target.")
-	if err != nil {
-		return err
-	}
+	// TODO: Validation of the org name removed. Proper validation
+	// of the targeted org is done by all the other commands using
+	// it anyway. If we really want it here and now, implement an
+	// `org show` command and API, and then use that API for the
+	// validation.
```

Comment:
Although an org can be deleted after being targeted (by another user for example), I think it is good UX to let the user know that the org she is targeting doesn't exist. Let's keep this node around until we have the `org show` endpoint.